tests/smoke.rs

```diff
@@ -109,7 +109,7 @@ fn test(
             cache_compressed,
             rafs_mode.parse().unwrap(),
             "api.sock".into(),
-            true,
+            !rafsv6,
```

Comment:
Can you please leave a comment either here or in the patch log for why digest validate for v6 is disabled.

Reply:
Actually already has some comments. exists only where it was first used.
```
            // FIXME: Currently no digest validation is implemented for rafs v6.
            !rafsv6,
```